.eslintrc.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
         'ecmaVersion': 12
     },
     'rules': {
-        'eqeqeq': 'error',
+        // 'eqeqeq': 'error',
+        'no-unused-vars': 'off',
```

Comment:
Haha, thinking out of the box. Nevis sakārtot reportētās problēmas, bet atslēgt problem reportingu. 💯 

MUCH AWESOME. SO WOW. **PART II.**

Tā gluži es nebiju domājis...

Reply:
Fake news. Sorry par anarhiju